docs/pages/install/sources.md

```diff
@@ -0,0 +1,178 @@
+# Stone Prover/Verifier Build Guide from Source
+This guide provides step-by-step instructions for building the Stone prover and verifier from source on Linux and macOS devices. The steps include setting up the environment, installing dependencies, and building the project using Bazelisk.
+
+### Prerequisites
+Ensure that your system meets the following dependencies before proceeding with the build process.
+
+- `Git`
+- `wget`
+- `Python 3.9+`
+- `pip` for Python package management
+
+## Install Dependencies
+### Linux
+```sh
+sudo apt-get update
+sudo apt-get install -y git wget python3 python3-pip
+```
+
+## Installation Guide
+### Step 1: Set Up Environment Variables
+To ensure compatibility with your system, set the `os` and `arch` variables by running the following commands:
+
+```sh
+os=$(uname | tr '[:upper:]' '[:lower:]')
+arch=$(uname -m | sed s/aarch64/arm64/ | sed s/x86_64/amd64/)
+```
+
+### Step 2: Set Environment Variables for Non-Interactive Installations
+This step prevents prompts during the installation of packages.
+```sh
+export DEBIAN_FRONTEND=noninteractive
+```
+
+### Step 3: Install System Dependencies
+Install the necessary system libraries using your os package manager.
+```sh
+sudo apt-get install -y libtinfo5 libdw-dev libgmp3-dev
+```
+
+### Step 4: Install Python Packages
+Install the required Python packages, including specific versions of sympy and cairo-lang, along with cpplint, pytest, and numpy.
+```sh
+pip install cpplint pytest numpy sympy==1.12.1 cairo-lang==0.12.0
+
+# If you encounter issues using pip due to PEP 668 https://peps.python.org/pep-0668/
+# - Alternative 1
+# Use a venv https://docs.python.org/3/library/venv.html
+# python3 -m venv .venv
+# source .venv/bin/activate
+# - Alternative 2
+# sudo rm /opt/homebrew/Cellar/python\@3*/**/EXTERNALLY-MANAGED
+# pip install cpplint pytest numpy sympy==1.12.1 cairo-lang==0.12.0 
+```
+
+### Step 5: Download and Set Up Bazelisk
+Download the Bazelisk binary appropriate for your OS and architecture.
+```sh
+wget "https://github.com/bazelbuild/bazelisk/releases/download/v1.20.0/bazelisk-$os-$arch"
```

Comment:
For linux the only suitable binary is https://github.com/bazelbuild/bazelisk/releases/download/v1.20.0/bazelisk-linux-amd64. That means you can simplify that